### DISCUSSION TITLE
Solana Governor Limits

### DISCUSSION THREAD

**Discussion by claudijd** [wormhole-foundation/wormhole#2922] [Governance] [answered]
I had a gut feeling that Solana outflows were not nearly as high as they had been historically and that there might be an opportunity to lower the current $50M/$500K Governor limit, as I have not seen outflows from Solana utilizing more than 10% of governor limits in a single day for some time now.

@kev1n-peters so graciously pulled the following, which are the top 10 token outflows from Solana since 4/10:

```
chain,address,symbol,name,notional (USD)
ethereum,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,USDC,USD Coin,$8,652,345
ethereum,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,WETH,Wrapped Ether,$3,276,042
solana,EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v,USDC,USD Coin,$3,253,451
solana,DUSTawucrTsGU8hcqRdHDCbuYhCPADMLM2VcCb8VnFnQ,DUST,DUST Protocol,$2,325,095
ethereum,0x6b175474e89094c44da98b954eedeac495271d0f,DAI,Dai Stablecoin,$1,727,600
ethereum,0xdac17f958d2ee523a2206206994597c13d831ec7,USDT,Tether USD,$1,006,994
solana,NFTUkR4u7wKxy9QLaX2TGvd9oZSWoMo4jqSJqdMb7Nk,BLOCK,Blockasset,$913,871
polygon,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,USDC,USD Coin (PoS),$396,964
solana,So11111111111111111111111111111111111111112,SOL,Wrapped SOL,$390,175
bsc,0x55d398326f99059ff775485246999027b3197955,USDT,Tether USD,$304,161
```

Summarizing an entire month of total flow outbound from Solana, it was a total of $22M in outflow during the month.  The current Governor limits would allow an entire two months of total flow of the top 10 tokens instantly in one day, which makes this feel like this could be lowered.

As a somewhat crude measure, I'd like the Guardians to consider reducing the Governor limit on Solana by 50% to $25M/$250K, which still feels overly generous given that you'd be able to move a month of organic volume in a single day with no delays.

**Comment by mcf-rocks** [comment]
doubt anyone is gonna argue against lowering a limit.

**Comment by brianlong** [comment, marked as answer]
I would argue with Mr. Smith, but he's a nice guy, and I agree with him. ;-) Lower the limit.

**Reply by claudijd** [reply]
PR: https://github.com/wormhole-foundation/wormhole/pull/2935
